### DISCUSSION TITLE
EJS + EJS-LAYOUTS Replacement?

### DISCUSSION THREAD

**Discussion by harshmandan** [eta-dev/eta#84] [Q&A]
**Is your feature request related to a problem? Please describe.**
Hello, I just run into your project and it looks and sounds amazing. But I have a doubt. I am currently using EJS + express-ejs-layouts combined. Can this be just dropped in and it will replace both the things, if yes how? I understand that I'll need to change the syntax a bit as well (adding `it` and `~` in all the places).

Thanks!

**Comment by shadowtime2000** [comment]
Hi @harshmandan,

I haven't used EJS before, mainly because I like handlebar syntax more (`{{}}`), but I think Eta should be able to replace both of those with [its partial support](https://eta.js.org/docs/learn/partials) which I believe is what `express-ejs-layouts` is kind of doing.

**Comment by harshmandan** [comment]
Okay so I'll give it a try and then let you know if it works. Thanks!
